Monika After Story/game/script-topics.rpy

```diff
@@ -8960,6 +8960,245 @@ label monika_hydration:
+                m 5tsu "Now that I remember... You know how to drive, dont't you?"
+                m 5hublb "I hope that you will take me to a lot of cool places!"
+            
+        "Not really.":
+            $ persistent._mas_likestravelling = False
+            m 1eka "Oww, that's okey, [player]~"
+            m 3huu "I wouldn't mind staying at home with you during vacations."
+return
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_desert",category=['misc'],prompt="Deserts",random=True))
+
+label monika_desert:
+    m 1eua "[player], have you ever been in a desert?"
+    m 1euu "I know this question is sudden, but it came out the blue."
+    m 3hsb "Did you know that deserts are quite unique in their own way? Although they don't offer a lot of positive factors."
+    m 4eub "Their temperatures vary between extreme hot during daytime and freezing cold at night."
+    m "In addition, their average rainfall is quite low, making living there a hard task for humans."
+    m 3eku "Still, their surface is a great spot for a solar power generation, plus oil is commonly found beneath all that sand, so deserts are not as bad as they seem."
+    m 1lku "Anyways...ehehe, I think I got carried a bit with all this information."
+    m 1esc "Just one more thing, [player]. If you ever travel to a desert don't forget to always carry water with you, alright?"
+    show monika 5lkc at t11 zorder MAS_MONIKA_Z with dissolve
+    m 5lkc "I wouldn't want to know that the person who I love the most suffers from extreme heat and dehydration in such a place."
+    m 5hub "Don't forget to take care of yourself, my love~"
+    return
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_metamorphosis",category=['literature, psychology'],prompt="The Metamorphosis",random=True))
+
+label monika_metamorphosis:
+    m 1eua "Hey [player], have you ever read {i}The Metamorphosis{/i}?"
+    m 4eku "It's a psychological novella that narrates the story of it's protagonist, who one day wakes up at morning, and finds himself transformed into a huge insect!"
+    m 4esu "The whole plot revolves around his daily situation, trying to adapt himself to the all the consequences of having such a new body."
+    m 3eku "And although it's not the happiest story, I would recommend you reading it. It's the type of books that really makes you think outside the box."
+    m 3eub "So if you ever have the time and chance to read it you should totally do it!"
+    m 3eua "Perhaps...we could even read it together."
+    m 1hublb "I don't mind reading it again if it's with you this time, my love~"
+    return
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_insecurities",category=['life, psychology, monika'],prompt="Insecurities",random=True))
+
+label monika_insecurities:
+    m 1lkc "Hey [player], I have been wondering..."
+    m 1ekc "This might sound weird but...do you feel grown up?"
```

Comment:
I edited the dialogue a bit. What do you think of it now?

    m 1lkc "Hey [player], I have been wondering..."
    m 1ekc "This might sound weird but...**do you feel accomplished in your life**?"
    [Deleted line]
    m 3ltc "You know, since I discovered that everything in my reality is just a composition of       code...I've been thinking quite a bit."